core/common/src/main/java/alluxio/master/metastore/rocks/RocksUtils.java

```diff
@@ -79,7 +79,7 @@ public static long readLong(byte[] bytes, int start) {
    *
    * @param <T> return type of parser's next method
    */
-  interface RocksIteratorParser<T> {
+  public interface RocksIteratorParser<T> {
```

Comment:
is this used?

Reply:
this is not used explicitly, but the compiler will complain on the call of `RocksUtils.createCloseableIterator` if it's not public